### PR TITLE
Fix the UI displaying name and systemMemoryContext allocationTag of OptimizedPartitionedOutputOperator

### PR DESCRIPTION
**Pull request by junyi1313**
Test plan - Running locally

```
== RELEASE NOTES ==

General Changes
* Fix the UI displaying name and systemMemoryContext allocationTag of OptimizedPartitionedOutputOperator
```

### CODE REVIEW

**Comment by junyi1313** [comment]
After this fix，presto ui can show the correct name of OptimizedPartitionedOutputOperator. @yingsu00 @zhenxiao Could you help take a look? Thanks.
image

**Review by zhenxiao — APPROVED** [review]
looks good
what do you think @yingsu00

**Review by kewang1024 — APPROVED** [review]
According to the [release guidance](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines)
Use "Fix" instead of "Correct"

**Comment by junyi1313** [comment]
> According to the [release guidance](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines)
> 
> 1. Use "Fix" instead of "Correct"
> 2. Can we rephrase it to be more user facing? Like from user's perspective, what is changed? eg: "Fix the UI displaying name of OptimizedPartitionedOutputOperator"
> 3. Besides 2,  does it also affect other aspects? I'm not an expert here but looks like it also affect memory accounting?

@kewang1024 
1 & 2. Sure, thanks for the reminder. The commit message and pr name have been updated, please check again. 
3. Yeah, before this change, the LocalMemoryContext allocationTag is PartitionedOutputOperator, which should be OptimizedPartitionedOutputOperator. So is also affect memory accounting.

**Comment by kewang1024** [comment]
> > According to the [release guidance](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines)
> > 
> > 1. Use "Fix" instead of "Correct"
> > 2. Can we rephrase it to be more user facing? Like from user's perspective, what is changed? eg: "Fix the UI displaying name of OptimizedPartitionedOutputOperator"
> > 3. Besides 2,  does it also affect other aspects? I'm not an expert here but looks like it also affect memory accounting?
> 
> @kewang1024 1 & 2. Sure, thanks for the reminder. The commit message and pr name have been updated, please check again. 3. Yeah, before this change, the LocalMemoryContext allocationTag is PartitionedOutputOperator, which should be OptimizedPartitionedOutputOperator. So it also affects memory accounting.

Thanks, looks good to me!

**Comment by zhenxiao** [comment]
thanks, @kewang1024 
I am rerunning it. Will merge once it is all successful

**Comment by junyi1313** [comment]
> thanks, @kewang1024 I am rerunning it. Will merge once it is all successful

@zhenxiao Thanks, all ci tests have passed. Please help to merge.
